src/components/GlobalErrorBoundary/__tests__/index.test.ts

```diff
@@ -29,9 +29,9 @@ describe('handleChunkError', () => {
   it('handles no sessionStorage value existence', () => {
     const isChunkError = handleChunkError(testChunkErrorObj)
 
-    expect(sessionStorage.getItem(LAST_CHUNK_FAILURE_RELOAD_KEY)).not.toBeNull()
-    expect(isChunkError).toBe(true)
-    expect(window.location.reload).toHaveBeenCalled()
+    expect(sessionStorage.getItem(LAST_CHUNK_FAILURE_RELOAD_KEY)).toBeNull()
+    expect(isChunkError).toBe(false)
+    expect(window.location.reload).not.toHaveBeenCalled()
```

Comment:
Interesting pivot 😬

Reply:
I realised it was a duplicate test that didn't fit the description 👀